src/analyze/analyze-plot.c

```diff
@@ -158,12 +160,33 @@ static void svg_graph_box(double height, double begin, double end) {
                             SCALE_Y * height);
         }
 }
+
+static void plot_tooltip(const char *name, char **ru[_UNIT_DEPENDENCY_MAX]) {
```

Comment:
This is called from only one place. So, I think it is OK to take `const UnitTimes *`.